### DISCUSSION TITLE
How can I reply to a tweet on mention?

### DISCUSSION THREAD

**Discussion by enbermudas** [tweepy/tweepy#1323] [Questions] [answered]
Is there a way to react to a mention (replying) as soon as it gets mentioned on a tweet?

**Comment by Harmon758** [comment, marked as answer]
You can use [statuses/filter](https://developer.twitter.com/en/docs/tweets/filter-realtime/api-reference/post-statuses-filter) [streaming](https://tweepy.readthedocs.io/en/latest/streaming_how_to.html) and track for the mention.
In your Stream Listener handler, you can use the the [POST statuses/update endpoint](https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/post-statuses-update) with [`API.update_status`](https://tweepy.readthedocs.io/en/latest/api.html#API.update_status) and its `in_reply_to_status_id` parameter to reply.
